crates/storage/provider/src/providers/blockchain_provider.rs

```diff
@@ -160,7 +160,7 @@ where
             // database lookup
             let Some(block_number) = provider.transaction_block(id)? else { return Ok(None) };
             let Some(body_index) = provider.block_body_indices(block_number)? else {
-                return Ok(None)
+                return Ok(None);
```

Comment:
please use nightly formatter, it should remove these semicolons

Reply:
I did run the nightly but it doesn't seem to care about the semicolons, will remove anyway

Reply:
ah yeah that's annoying -.- thank you!